core/src/main/scala/akka/persistence/cassandra/CassandraStatements.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.cassandra
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+import akka.Done
+import akka.annotation.InternalApi
+import akka.persistence.cassandra.journal.CassandraJournalConfig
+import akka.persistence.cassandra.journal.CassandraJournalStatements
+import akka.persistence.cassandra.snapshot.CassandraSnapshotStatements
+import akka.persistence.cassandra.snapshot.CassandraSnapshotStoreConfig
+import com.datastax.oss.driver.api.core.CqlSession
+
+@InternalApi
+private[akka] trait CassandraStatements {
```

Comment:
had to do this refactoring of the statements because same `init = executeAllCreateKeyspaceAndTables` must be from all 3 places

Reply:
is there a way to disable the snapshot statements? I could imagine ppl uisng this and not the snapshot store

Reply:
i see they have separate autocreates

We should probably set auto-create to false by default in 1.0

Reply:
I agree, created issue https://github.com/akka/akka-persistence-cassandra/issues/645